io.emeric.toolblex.yml

```diff
@@ -0,0 +1,260 @@
+app-id: io.emeric.toolblex
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
```

Comment:
```suggestion
runtime: org.kde.Platform
runtime-version: '6.4'
sdk: org.kde.Sdk
```

And don't build Qt.

Reply:
I understand, but that's really not what I want to do. I want control of what I'm using, and that mean specifying a precise version of Qt. Also it must be pretty new, it's the first time I'm seeing this one.

There are many bugs and regressions depending on what Qt version I'm using, and there is a whole lot of new features included in each new Qt version, that I usually end up adopting sooner than later.

For instance, for the other app I have on flathub (io.emeric.watchflower) there is a segfault with Qt 6.4.0+, so I'm sticking with 6.3.1 until the next app version.
For the next version of this app (io.emeric.toolblex) I have a set of new features entirely based on Qt 6.5 that I'll need.

Also, I don't need anything in the KDE repo, just Qt. Qt apps like mine are not KDE apps. It would be nice to have a Qt only SDK for flatpak (per Qt major version).

Reply:
6.3 is in the 6.3 version of the runtime, so that's not an issue.

Reply:
Yes, but there wasn't a KDE 6.3 SDK when I released that app ;)